src/main/java/org/jabref/preferences/JabRefPreferences.java

```diff
@@ -354,7 +354,7 @@ public class JabRefPreferences implements PreferencesService {
     public static final String OO_SHOW_PANEL = "showOOPanel";
     public static final String OO_SYNC_WHEN_CITING = "syncOOWhenCiting";
     public static final String OO_USE_ALL_OPEN_BASES = "useAllOpenBases";
-    public static final String OO_BIBLIOGRAPHY_STYLE_FILE = "ooBibliographyStyleFile";
+    public static final String JSTYLE_FILE = "jStyleFile";
```

Comment:
Please do not rename the preferences, as otherwise existing users will have problems. You would need to provide a migration otherwise

Reply:
Reverted